backend/note_taking/views/block_views.py

```
@@ -0,0 +1,68 @@
+        return Block.objects.filter(note__notebook__owner=self.request.user)
+
+class DeleteBlockEndpoint(DestroyAPIView):
+    serializer_class = BlockSerializer
+    permission_classes = [IsAuthenticated]
+
+    def get_queryset(self):
+        return Block.objects.filter(note__notebook__owner=self.request.user)
+
+    def perform_destroy(self, instance):
+        if instance.note.notebook.owner.id != self.request.user.id:
+            raise PermissionDenied("You do not have permission to delete blocks in this note")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)